### DISCUSSION TITLE
Thunderbolt support NUC10i5

### DISCUSSION THREAD

**Discussion by pierpaolodimarzo** [osy86/HaC-Mini#572] [General]
Hi man, 

I need your great help .. I just finished setting the EFI for my NUC10i5 ... it was tough but I did it ... I have a big problem with the TB part though; 
the NUC mounts the Intel JHL7540 Titan Ridge 4C 2018, but I can't get it to read under "About this MAC - Thunderbolt"

Could you help me?

Thank you very much .. you would be very valuable to my project. It would be a valuable collaboration to develop the NUC10

**Comment by osy** [comment]
Please use the discussion tab. I have no idea what your problem is as there are 1,000 things that could have gone wrong. I'll leave this thread open for others to discuss but I don't provide any support for non Hades Canyon builds.
